### PR TITLE
Ci/add job feature

### PR DESCRIPTION
**Pull request by stebenoist**
# Pull Request description

## Description of the change
Add CI tests around job

### What I did
- Add CI tests around job
- Allow to execute hosted operations on bootstrapped Yorc

### CODE REVIEW

File: testdata/ci/features/Job.feature [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@Job
+Feature: Deploy a TestJobApp application using Alien4Cloud with a job mock
+
+  Background: Deploying required artifacts
+    Given I have uploaded the artifact named "org-ystia-samples-jobmock" to Alien
+    And I have built the artifact named "testJob" from templates named "testJob" to Alien
+    And I have uploaded the artifact named "testJob" to Alien
+
+  @CI
+  @hp
```
Comment (loicalbertin) [inline testdata/ci/features/Job.feature:14]: HostsPool only?

File: testdata/ci/features_alien_test.go [hunk 2]
```diff
@@ -209,6 +209,70 @@ func (c *suiteContext) iRunTheWorkflowNamed(workflowName string) error {
 	return nil
 }
 
+func (c *suiteContext) iRunAsyncTheWorkflowNamed(workflowName string) error {
+	if c.applicationID == "" || c.environmentID == "" {
+		return fmt.Errorf("Missing mandatory context parameter applicationID: %q or environmentID: %q", c.applicationID, c.environmentID)
+	}
+
+	go func() {
```
Comment (loicalbertin) [inline testdata/ci/features_alien_test.go:217]: Running a routine without a way stop it is a bad practice :smile:

File: testdata/ci/go.mod [hunk 3]
```diff
@@ -4,10 +4,10 @@ go 1.13
...
-	github.com/alien4cloud/alien4cloud-go-client/v2 v2.2.0-M2.0.20200130102046-9e00049b456c
-	github.com/chromedp/cdproto v0.0.0-20191114225735-6626966fbae4
-	github.com/chromedp/chromedp v0.5.2
+	github.com/PuerkitoBio/goquery v1.5.1
+	github.com/alien4cloud/alien4cloud-go-client/v2 v2.2.0-milestone.6
+	github.com/chromedp/cdproto v0.0.0-20200209033844-7e00b02ea7d2
+	github.com/chromedp/chromedp v0.5.3
 	github.com/spf13/viper v1.6.1
-	golang.org/x/sys v0.0.0-20200103143344-a1369afcdac7 // indirect
+	github.com/ystia/yorc/v4 v4.0.0-rc.1
```
Comment (loicalbertin) [inline testdata/ci/go.mod:12]: Why do we need yorc there?

File: testdata/ci/features/Job.feature (hunk 1 above)
Comment (stebenoist) [reply testdata/ci/features/Job.feature:14]: Yes but it can be extended for all infras

File: testdata/ci/go.mod (hunk 3 above)
Comment (stebenoist) [reply testdata/ci/go.mod:12]: seems weird. I remove it

File: testdata/ci/features_alien_test.go (hunk 2 above)
Comment (stebenoist) [reply testdata/ci/features_alien_test.go:219]: fixed

File: testdata/ci/features/Job.feature (hunk 1 above)
Comment (stebenoist) [reply testdata/ci/features/Job.feature]: Done

File: testdata/ci/features_alien_test.go (hunk 2 above)
Comment (loicalbertin) [reply testdata/ci/features_alien_test.go:219]: Hum I think your first through was better.
I didn't realize that RunWorkflow was synchronous and its cancellation done thanks to the context.
So as soon RunWorkflow returns for completion or cancellation then we will exit the routine.
Sorry about this.

Comment (stebenoist) [reply testdata/ci/features_alien_test.go:217]: done. But why the previous version was better ?

**Pull request merged**